graphviz2drawio/mx/CurveFactory.py

```diff
@@ -0,0 +1,21 @@
+from svg.path import CubicBezier
+from svg.path import parse_path
+from .Curve import Curve
+
+
+class CurveFactory:
+    def __init__(self, coords):
+        super(CurveFactory, self).__init__()
```

Comment:
@jcaplan agree that it's unnecessary in the current implementation, but my understanding is that it's Python best practice. See https://stackoverflow.com/questions/6796996/call-super-init-in-classes-derived-from-object Thoughts?

Reply:
I lean towards being sparse and clear over being consistent. If I need it later, I'll add it later. I think 9/10 times you don't wind up needing it later, and over time you get a lot of padding in the code which can make it less clear.

I was just watching this video on Java patterns in Python:
https://www.youtube.com/watch?v=Bd2Mn9AmVNA&t=969s

The theme applies to my comment in general about the factory pattern. There's nothing wrong with using it, but if you're not reusing the same factory object over and over again, and if there's nothing expensive being reused, and if there's not a lot of global state in the factory object which a lot of different actors need to touch, then it's not doing much.

I guess I also really like mixing and matching OOP and procedural in python. both are allowed, and it's very nice to do whichever makes more sense for each part of the problem. I think aesthetic consistency is not worth the trouble compared to conciseness and simplicity.

Another arguably java-esque pattern that isn't serving clarity is the one class per file. There are 83 LOC (not counting blanks) in models subpackage in 7 files.  This could easily be consolidated to one file models.py without sacrificing any clarity. Similarly 439 LOC in 9 files in mx. Could probably be reduced to a few files.

It's a bit too much architecture and for someone who's coming in trying to reverse engineer it makes things quite a bit harder. And I think that's a worthwhile design principle for open source.

Anyway this is very subjective and it's your call but that's my 2 cents.